Client/Coordinators/QRCode/QRCodeCoordinator.swift

```diff
@@ -35,6 +35,5 @@ class QRCodeCoordinator: BaseCoordinator, QRCodeDismissHandler {
 
     func dismiss(_ completion: (() -> Void)?) {
         router.dismiss(animated: true, completion: completion)
-        parentCoordinator?.didFinish(from: self)
```

Comment:
@lmarceau removing this call to `parentCoordinator?.didFinish(from: self)` because dismisses the Tab tray too soon, after QRCodeVC is dismissed we need to push `FxAWebViewController`. I'm afraid this change will break other QRCode flows checking if you have any ideas on how to handle it properly